core/settings/base.py

```diff
@@ -148,3 +148,4 @@
 
 # Hashid settings
 HASHID_FIELD_SALT = dotenv.get('HASHID_FIELD_SALT')
+DEBUG = dotenv.get('DEBUG')
```

Comment:
@andela-ooshodi: Yes, DEBUG is set to false in production and true in staging on Heroku.